### PR TITLE
Refactor draw components

### PR DESCRIPTION
**Pull request by maryangela**
Refactoring of the DrawComponents class was done and the up, down, left and right methods of the pacman class were modified

### CODE REVIEW

**Comment by samuelloza** [comment]
Please change **master** to **dev**

File: src/test/java/pacman/PacmanTest.java [hunk 1]
```diff
@@ -5,25 +5,29 @@
 	
 	@Test
 	public void testMoveUp(){
-		Pacman pacman = new Pacman(60, 60, true);
-		assertEquals(30, pacman.up());
+        Pacman pacman = new Pacman(60, 60, true);
+        pacman.up();
+		assertEquals(30, pacman.getY());
```
Comment (JhordanSoto) [inline src/test/java/pacman/PacmanTest.java]: "assertEquals(30, pacman.getY());"  has extra space please change this

File: src/test/java/pacman/PacmanTest.java [hunk 2]
```diff
@@ -5,25 +5,29 @@
...
 	}
 	
 	@Test
 	public void testMoveDown(){
-		Pacman pacman = new Pacman(60, 60, true);
-		assertEquals(90, pacman.down());
+        Pacman pacman = new Pacman(60, 60, true);
+        pacman.down();
+		assertEquals(90, pacman.getY());
```
Comment (JhordanSoto) [inline src/test/java/pacman/PacmanTest.java]: the same here

**Comment by samuelloza** [comment]
@maryangela  thanks for pushing this PR, but it's has conflicts, please check  it and push another commit to fix them, thanks!

**Pull request merged**